api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/controllers/handlers/NotFoundErrorController.java

```diff
@@ -26,7 +25,7 @@
  */
 @Controller
 @Order(Ordered.HIGHEST_PRECEDENCE)
-public class NotFoundErrorController implements ApimlErrorController {
+public class NotFoundErrorController {
 
 
     private static final String PATH = "/not_found";    // NOSONAR
```

Comment:
This is now unused

Reply:
it is used, there is an internal call when URL is not found, the interface was not used

Reply:
Pablo meant the field - removed